regulations/templates/regulations/tree/preamble_intro.html

```diff
@@ -4,15 +4,30 @@
 
 <li
     {% if node.marker %}marker-type="{{node.marker}}"{% endif %}
-    id="{{node.markup_id}}" data-permalink-section
->
+    {% if node.toc_id %}data-toc-id="{{node.toc_id}}"{% endif %}
+    id="{{node.markup_id}}"
+    data-permalink-section>
+  <div class="node">
+    {% if node.accepts_comments %}
+      <div
+          class="activate-write"
+          data-section="{{ node.full_id }}"
+          data-label="{{ node.human_label }}">
+        <a href="#">
+          <div class="paragraph-comment-icon">
+            <span class="fa fa-pencil-square-o" aria-hidden="true"></span>
+          </div>
+          <div class="paragraph-comment">Write a comment about {{ node.human_label }}</div>
+        </a>
+      </div>
+    {% endif %}
```

Comment:
I don't think we _want_ to allow users to comment here?
<img width="966" alt="screen shot 2016-05-10 at 10 46 14 am" src="https://cloud.githubusercontent.com/assets/326918/15150737/727a9902-169c-11e6-84e0-39e386458b86.png">